3.0/adaptivesvc-triple/client/main.go

```diff
@@ -72,6 +72,8 @@ func main() {
 					logger.Infof("Reach Limitation")
 				} else if err.Error() == context.DeadlineExceeded.Error() {
 					logger.Warnf("Consumer Request Timeout, err: %v", err)
+				} else if offline_simulator.IsServerOfflineErr(err) {
+					logger.Warnf("Server offline, err: %v", err)
```

Comment:
I think catching errors, e.g. reach limitation, timeout, etc., could encapsulate a method, so that it could easily to apply to other test functions, like `sleep`.